webpack.config.js

```diff
@@ -77,14 +78,14 @@ const babelLoader = {
 };
 
 const webpackConfig = {
-	bail: calypsoEnv !== 'development',
+	bail: ! isDevelopment,
 	entry: {},
 	devtool: 'false',
 	output: {
 		path: path.join( __dirname, 'public' ),
 		publicPath: '/calypso/',
-		filename: '[name].[chunkhash].js', // prefer the chunkhash, which depends on the chunk, not the entire build
-		chunkFilename: '[name].[chunkhash].js', // ditto
+		filename: '[name].[chunkhash].min.js', // prefer the chunkhash, which depends on the chunk, not the entire build
```

Comment:
Is it still necessary to have the `.min.js` suffixes? Wouldn't just `.js` be equally good?

Reply:
+1 on ditching the min's if we can.

Reply:
i used to be of the opinion that `.min.js` was unimportant and merely a side-effect of our 2-step build system -- but after you found a bug just based on filename I realized it can be helpful to know at-a-glance if a file should be minified or not.

so the `.min.js` extensions grew on me over time.  do you have any reasoning why `.js` by itself is better?

Reply:
Using just `.js` looks like the natural default -- we no longer have two versions of the file, but Webpack outputs just one, already minified.

If you want to keep the `.min.js` for the sake of clarity, I don't mind -- it doesn't really matter much.